app.js

```diff
@@ -1,22 +1,21 @@
-/*eslint no-undef: "error"*/
-/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
 /*eslint-env node*/
+
 const express = require('express');
 const app = express();
-
-const config = require('config');
-const dbConfig = config.get('Item.dbConfig');
-
 const bodyParser = require('body-parser');
+const passport = require('passport');
+const session = require('express-session');
+
+app.use(session({ secret: 'keyboard cat', resave: false, saveUninitialized: true }));
+app.use(passport.initialize());
+app.use(passport.session());
+app.use(require('morgan')('combined'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
-
 app.use(express.static('public'));
 
-app.use('/', require('./routes/'));
-
-app.listen(dbConfig, () => {
-	// Port listen to Port 4002
-});
+app.use('/', require('./routes/items.js'));
```

Comment:
oh, I would still prefer to have `./routes/index.js` which should call individual route modules.
actually the good practice is the following:
* /index.js which imports src/app.js and creates a server by calling `listen`
* /src which holds all other source files
  * app.js - root module for your app. calls initialization modules and root route module from /src/routes/index.js
  * /routes - which holds all routes
    * index.js - root module which calls all routes
    * route-name.js - route handler for individual routes. should be as simple as possible, logic should be limited to validation of input only. the purpose is handling transport (REST over HTTP)
  * /model (or /controllers) - holds modules to handle the business logic, regardless of the transport (we don't care if request to alter model entities comes from REST or by some other means)
    * model-entity.js - a module which handles model entity (say, user or item). typically provides CRUD operations or querying facilities
  * /tests - holds all the tests
    * /unit - holds the tests for individual functions in model modules, dependencies should be mocked. That should be the majority of your tests
      * model-entity.spec.js - tests for model entity from /src/model/model-entity.js
    * /integration - holds the tests for intermodule interaction, could be driven through the transport layer (i.e. supertest + app instance in our case)

You can extend this model when you add extra layers to your app (say, web or mobile client) and still keep the clarity of the structure.

Reply:
I have moved, renamed and exported/imported functions. 
I will need add unit tests.